tox.ini

```diff
@@ -38,7 +38,7 @@ commands =
     rm tests/__init__.py
     pycodestyle help_tokens tests
     pydocstyle help_tokens tests
-    python setup.py check --restructuredtext --strict
```

Comment:
`readme-renderer==35.0` dropped the legacy `check` command and suggests to switch using `twine` instead. https://github.com/pypa/readme_renderer/blob/main/CHANGES.rst#350-2022-04-19